src/enforcement.c

```diff
@@ -44,7 +44,7 @@ init_disk_quota_enforcement(void)
 static bool
 quota_check_ExecCheckRTPerms(List *rangeTable, bool ereport_on_violation)
 {
-	ListCell *l;
+	ListCell *volatile l;
```

Comment:
Why does this need to be done?

Reply:
The compiler throws a warning for this variable too.